doc/news/_preparation_next_release.md

```diff
@@ -175,8 +175,9 @@ These notes are of interest for people developing Elektra:
 - `clang-5.0` is now used for clang tests by the build system *(Lukas Winkler)*
 - An additional build job on Ubuntu:xenial has been added *(Lukas Winkler)*
 - Several improvments to the build system have been implemented *(Lukas Winkler)*:
-  - Better Docker image handling
-  - abort of previously queued but unfinished runs on new commits
+  - Better Docker image handling.
+  - Abort of previously queued but unfinished runs on new commits.
+  - Document how to locally replicate the Docker environment used for tests.
```

Comment:
Please add your name

Reply:
It is a sub bulletin of an entry that has my name. But apparently that is not enough for the current script.

Reply:
Ahh, it is also not enough for me: I did not see it, too.

Thus you did not change any code, we can merge this even without the okay from the build server.

Please set "ready to merge" when you fixed everything from the review.